gen/edu/uc/emiller/carapp/dto/Car.kt

```diff
@@ -0,0 +1,29 @@
+package edu.uc.emiller.carapp.dto
+
+import androidx.room.*
+
+/**
+ * The Car DTO.
+ *
+ * Stores the make, model and year of a car.
+ * @param make the brand of the car
+ * @param model the model or series of the car
+ * @param year the year of the series
+ */
+@Entity(tableName = "car")
+data class Car(
+        @ColumnInfo(name = "make")
+        var make: String = "",
+
+        @ColumnInfo(name = "model")
+        var model: String = "",
+
+        @ColumnInfo(name = "year")
+        var year: Int = 0,
+
+        @PrimaryKey var car_id: Int = 0
+) {
+    override fun toString(): String {
+        return "$year $make $model"
+    }
+}
```

Comment:
Basic layout of a DTO that is configured for use with Room DB. Pretty much identical to individual assignment or what you would see in MyPlantPlaces, though I did use the @ColumnInfo attribute.